app/controllers/admin/attachments_controller.rb

```diff
@@ -44,13 +44,13 @@ def update_many
     params[:attachments].each do |id, attributes|
       attachment = attachable.attachments.find(id)
       attachment.assign_attributes(attributes.permit(:title))
-      unless attachment.save(context: :user_input)
+      if attachment.save(context: :user_input)
+        ServiceListeners::AttachmentUpdater.call(attachment_data: attachment.attachment_data)
```

Comment:
This looks good! I think it could probably be applied to the destroy method as well, rather than calling `attachment_updater`